t/selenium.js

```diff
@@ -305,15 +305,15 @@ async function writePreviousSeleniumCoverage() {
   }
 }
 
-async function handleCommandAndWait({command, file, target, value}, t) {
+async function handleCommandAndWait({command, target, value}, t) {
   const newCommand = command.replace(/AndWait$/, '');
 
   const html = await findElement('css=html');
-  await handleCommand({command: newCommand, file, target, value}, t);
+  await handleCommand({command: newCommand, target, value}, t);
   return driver.wait(until.stalenessOf(html), 30000);
 }
 
-async function handleCommand({command, file, target, value}, t) {
+async function handleCommand({command, target, value}, t) {
```

Comment:
Good catch, `file` should likely have been removed with https://github.com/metabrainz/musicbrainz-server/commit/f5dc19259fef74ac3905314d822307bef48fc726#diff-df0bc9d369c9ac178ace161912b5d7880f997594ad349f3ec7f67e4c918ffab5L301-L319